README.md

```diff
@@ -1,7 +1,24 @@
-Airport Challenge
-=================
+| Objects | Properties                  | Messages                            | Outputs  |
+| ------- | --------------------------- | ----------------------------------- | -------- |
+| Airport | landedPlanes @Array[@Plane] | landPlane()(@Plane)                 | @Void    |
+|         | airportCapacity @Integer    | getAirportCapacity()                | @Integer |
+|         |                             | increaseAirportCapacityTo(@Integer) | @Void    |
+|         |                             | isAirportFull()                     | @Boolean |
+|         |                             | isPlaneAtTheAirport()               | @boolean |
+|         |                             | checkWeather()                      | @String  |
+| Plane   | id @String                  | getId()                             | @String  |
 
-```
+## 1. Clone to your system
+
+## 2. In terminal type node specRunner.js
+
+## 3. run multiple tests because of test number 7 and 8 (weather landing check)
+
+## 4. With the exception of test number 7 and 8, all tests should pass on every run
+
+# Airport Challenge
+
+``````
          ______
```

Comment:
For some reason my airport.js file starts with a small a, even though the same files on my pc start with capital letter.